src/select/src/components/CheckboxSelect.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+import Select from 'react-select';
+
+import Option from './styled/CheckboxOption';
+
+import withProps from '../../../components/withProps';
+import selectStyles from '../../../style/selectStyles';
+
+export const props = {
+  styles: selectStyles,
+  menuPlacement: 'auto',
+  components: {
+    Option
+  },
+  isMulti: true,
+  hideSelectedOptions: false,
+  closeMenuOnSelect: false,
+};
+
+const CheckboxSelect = withProps(Select, props);
+CheckboxSelect.displayName = 'CheckboxSelect';
```

Comment:
This is a `styled-components` thing, right? Can you add `displayName` to the other components as well?

Reply:
This is a React thing. By default, the displayName of components will be the same as the component. The `babel-plugin-styled-components` handles the `styled-components`.

However, since CheckboxSelect is using the `withProps` HOC, it was being displayed as `LatticeWrapper`. This was added so I could use [enzyme's display name selector](https://airbnb.io/enzyme/docs/api/ReactWrapper/find.html) instead of having to import each component into the test file.

I can add `displayNames` to the other Lattice wrapped `Select` and `Creatable` components, but what would I name them so they don't conflict with `react-select` display names?

Any suggestions? `OLSelect`/`OLCreatable`? Maybe just call them dropdowns?